SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/GridBasedLBStrategy.h

```diff
@@ -26,9 +31,9 @@ class SPATIALGDK_API UGridBasedLBStrategy : public UAbstractLBStrategy
 	UGridBasedLBStrategy();
 
 /* UAbstractLBStrategy Interface */
-	virtual void Init(const class USpatialNetDriver* InNetDriver) override;
+	virtual void Init(const USpatialNetDriver* InNetDriver) override;
 
-	virtual TSet<VirtualWorkerId> GetVirtualWorkerIds() const;
+	virtual TSet<VirtualWorkerId> GetVirtualWorkerIds() const override;
```

Comment:
Can we return const TSet<VirtualWorkerId>&?

Reply:
I don't think so. I believe we create and return a new set from an array.